### DISCUSSION TITLE
能否主动添加好友？

### DISCUSSION THREAD

**Discussion by qingyiwebt** [howmanybots/onebot#153] [想法 / 建议]
最近想基于onebot编写一个第三方的QQ客户端，但是我发现并没有提供主动添加好友的Api。

**Comment by ExerciseBook** [comment]
不太能提供这个。我大概建议这样操作。

1. 自己偷偷写好，不要大规模公开。
2. 如果要加好友，建议用手机加。加好友加群啥的属于高风险操作。

主要是担心这个版本的 QQ 协议实现被 abuse 之后，容易被腾讯精准打击。
